packages/inventory-compliance/src/SystemRulesTable/SystemRulesTable.js

```diff
@@ -158,7 +157,7 @@ class SystemRulesTable extends React.Component {
         )
     )
 
-    selectedRules = (rules) => {
+    addIsSelected = (rules) => {
```

Comment:
This function was a bit misleadingly named, and it conflicted with one I wanted to add.